### PR TITLE
feat: allow changing the default revisionHistoryLimit

### PR DESCRIPTION
**Pull request by tberreis**
**Description of the change**

The deployment has been adapted to allow the user changing the default revisionHistoryLimit.

**Benefits**

You may want to clean up old replicasets so that these don't appear in metrics, OPA violations, ArgoCD UI and so on.
Therefore you want to overwrite the Kubernetes default revision history limit.

**Possible drawbacks**

N/A

**Applicable issues**

N/A

**Additional information**

N/A

### CODE REVIEW

File: helm/sealed-secrets/values.yaml [hunk 1]
```diff
@@ -12,6 +12,10 @@ fullnameOverride: ""
 ## @param namespace Namespace where to deploy the Sealed Secrets controller
 ##
 namespace: ""
+## @param revisionHistoryLimit Number of old history to retain to allow rollback (If not set, default Kubernetes value is set to 10)
+## e.g:
+## revisionHistoryLimit: 1
```
Comment (alemorcuq) [inline helm/sealed-secrets/values.yaml:17]:
```suggestion
revisionHistoryLimit: ""
```

I would also move this a bit down, under the "Sealed Secrets Parameters" section.

Could you please add this parameter to the readme too?

**Review by alemorcuq — CHANGES_REQUESTED** [review]
Thanks for your contribution, @tberreis ! Please, check my comment.

File: helm/sealed-secrets/values.yaml (hunk 1 above)
Comment (tberreis) [reply helm/sealed-secrets/values.yaml]: Thank you for your feedback @alemorcuq. Code has been adapted.

**Review by alemorcuq — APPROVED** [review]
Thanks!

**Pull request merged**